DQM/SiStripMonitorCluster/python/SiStripMonitorCluster_cfi.py

```diff
@@ -271,6 +271,19 @@
         globalswitchon = cms.bool(True)
         ),
 
+    ClusWidthVsAmpTH2 = cms.PSet(
+        Nbinsx = cms.int32(2000),
+        xmin = cms.double(0.),
+        xmax = cms.double(2000.),
+        Nbinsy = cms.int32(100),
+        ymin = cms.double(0.),
+        ymax = cms.double(100.),
+        globalswitchon = cms.bool(False),
+        subdetswitchon = cms.bool(True), 
+        layerswitchon  = cms.bool(True),
+        moduleswitchon = cms.bool(False)
```

Comment:
@hdelanno if we set only the globalswitchon to True and others to False, we keep a single plot in the DQM, still enough for routine checks and if need be, one can reprocess privately with all plots alive. Could you please set subdet/layer to False and global to True? That should save a lot of space.
Sorry for my delayed comment.